packages/ti_eset/data_stream/apt/manifest.yml

```
@@ -0,0 +1,85 @@
+title: "APT"
+type: logs
+streams:
+  - input: httpjson
+    title: APT
+    description: Collect data from EI APT feed
```

```suggestion
    description: Collect data from ETI APT feed
```
Is this supposed to be ETI? If so, similarly for other datastreams.